keys.txt

```diff
@@ -602,6 +602,12 @@ military
 community_centre
 bank
 operational // trolltag referring to current POI
+users_PLWD // yes, mapping it in OSM is highly dubious
+users_boy // yes, mapping it in OSM is highly dubious
+users_elderly // yes, mapping it in OSM is highly dubious
+users_female // yes, mapping it in OSM is highly dubious
+users_girl // yes, mapping it in OSM is highly dubious
```

Comment:
https://www.openstreetmap.org/changeset/158333852

Reply:
thanks @matkoniecz ; merged (and simplified those multiple related keys into regex)